amm/src/main/scala/ammonite/main/TrapExitSecurityManager.scala

```diff
@@ -0,0 +1,18 @@
+package ammonite.main
+import java.security.Permission
+
+/**
+ * This security manager is assigned as default one when we run user scripts with the --watch option
+ * This allows to trap sys.exit by throwing a special TrapExitException
+ */
+object TrapExitSecurityManager extends SecurityManager {
```

Comment:
Can this object be `private [ammonite]`?

Reply:
AFAIK, little effort has been made to clean up the public API of this module, which is not really meant to be used as a library anyway. So it shouldn't be a problem to have this be public.